tests/integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -9,6 +9,7 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
+	[ "${KATA_HYPERVISOR}" == "cloud-hypervisor" ] && skip "test not working"
```

Comment:
Could you raise an issue about these tests not working and then add a link to the issue(s) in these `skips`.